### PR TITLE
add a GPL 3 licence

### PR DESCRIPTION
**Pull request by remi-dupre**

### CODE REVIEW

**Comment by seirl** [comment]
I rebased this PR on master.

File: COPYING [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (seirl) [inline COPYING:1]: This is already present in markdown format, please remove it

File: prologin/contest/staff_views.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Copyright (C) <2015> Association Prologin <association@prologin.org>
```
Comment (seirl) [inline prologin/contest/staff_views.py:1]: Year shouldn't be inside `<>`

**Review by seirl — CHANGES_REQUESTED** [review]

**Comment by juli0z** [comment]
Migrated to https://gitlab.com/prologin/concours/site/-/merge_requests/216

**Pull request closed without merging**